pkg/virt-handler/vm.go

```diff
@@ -899,6 +899,9 @@ func (d *VirtualMachineController) updateVolumeStatusesFromDomain(vmi *v1.Virtua
 	if len(vmi.Status.VolumeStatus) > 0 {
 		diskDeviceMap := make(map[string]string)
 		for _, disk := range domain.Spec.Devices.Disks {
+			if disk.Alias == nil {
+				continue
+			}
```

Comment:
This is unlikely to be the proper fix to the issue as other places in the codebase will consider `Alias` to be set.